api/src/extensions/manager.ts

```diff
@@ -212,7 +213,7 @@ export class ExtensionManager {
 		this.messenger.subscribe(this.reloadChannel, (payload: Record<string, unknown>) => {
 			// Ignore requests for reloading that were published by the current process
 			if (isPlainObject(payload) && 'origin' in payload && payload['origin'] === this.processId) return;
-			this.reload();
+			this.reload({ forceSync: true });
```

Comment:
Do we still need this?

Reply:
Per discussion this works as it disregards the status file check which as [pointed out](https://github.com/directus/directus/issues/25270#issuecomment-2936456482) is not cleared after initialization. For the case where the volume is not shared after the pods initially boot up they will never redownload any new extensions from the location as the status file will always indicate done. 

If we use forceSync this will end up being a performance issue as it will be redownloading the extensions on every reload event.


Most likely we will need to utilize a hash in the status file and compare that to determine if the extensions should be synced down again during a reload event.